File: src/main/java/fr/lewon/web/bot/entities/input/leagues/ActionLeaguesGetOpponentInfo.java
package fr.lewon.web.bot.entities.input.leagues;

import fr.lewon.bot.http.body.urlencoded.FUEMember;

public class ActionLeaguesGetOpponentInfo extends ActionLeagues {

	@FUEMember("opponent_id")
	private String opponentId;

	public ActionLeaguesGetOpponentInfo(String opponentId) {
		super("get_opponent_info");
		this.opponentId = opponentId;
	}

	public String getOpponentId() {
		return opponentId;
	}

	public void setOpponentId(String opponentId) {
		this.opponentId = opponentId;
	}

}

File: src/main/kotlin/fr/lewon/bot/hh/entities/input/leagues/ActionLeaguesGetOpponentInfo.kt
package fr.lewon.bot.hh.entities.input.leagues

import com.fasterxml.jackson.annotation.JsonProperty

class ActionLeaguesGetOpponentInfo(opponentId: String) : ActionLeagues("get_opponent_info") {

    @JsonProperty("opponent_id")
    val opponentId: String = opponentId

}